HW_7/task3.py

```diff
@@ -0,0 +1,8 @@
+a = "Hello"
+result = {}
+for i in a:
+    if i not in result:
+        result[i] = 0
+    result[i] = result[i] + 1
+print(result)
+
```

Comment:
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result